components/tests/ui/robot_setup.sh

```diff
@@ -57,6 +68,13 @@ do
   bin/omero import -d $delDs $TINY_IMAGE_NAME --debug ERROR
 done
 
+# Import Plate
+bin/omero import $PLATE_NAME --debug ERROR > plate_import.log 2>&1
+plateid=$(sed -n -e 's/^Plate://p' plate_import.log)
+# Use populate_metadata to upload and attach bulk annotation csv
+PYTHONPATH=$PYTHONPATH:lib/python
+python lib/python/omero/util/populate_metadata.py -k $key Plate:$plateid $BULK_ANNOTATION_CSV
```

Comment:
f12a950 should help, but that would require pulling in the rest of that branch. The point of the spaces is to prevent us from needing to prematurely merge everything. If all the test runs are transferred to the other build, then the theory is when we open metadata --> develop, things should remain green.